### DISCUSSION TITLE
如何更优雅地搭乘公共交通

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#34] [Announcements]
# 如何更优雅地搭乘公共交通

主流城市公交地铁系统已经全面支持 NFC，但未见大规模推广，目前普遍还是在宣传二维码，但一旦用过 NFC，就再也回不去了。

https://zexwoo.blog/posts/tutorials/nfc/

**Comment by abc1763613206** [comment]
交通联合卡可用地区查询： https://ipt.kopisee.com/canton/zh-cn/t-union
交联面条站的收单记录查询： https://www.tunionfans.com/record/
